readme.md

```diff
@@ -10,6 +10,14 @@ The application architecture uses the following AWS services.
 * [S3](https://aws.amazon.com/s3/) - Object store for hosting frontend.
 * [Cognito](https://aws.amazon.com/cognito/) - user management, authentication, and authorization.
 
+## Table of Contents
+
+1) [Frontend](./01-frontend.md)
+1) [User Management](./02-user-management.md)
+1) [Environment Parameters & Secrets](./03-environment-parameters.md)
+1) [Backend API](./04-backend-api.md)
+1) [Production Deployment](./05-production.md)
```

Comment:
I feel like there should be some explanation that there are setup steps below that need to be completed first if you don't already have an AWS and Stackery account.

Then, at the bottom of the setup steps, there should also be a link to the first module so people know where to go next. Actually, there should probably be a link to the next module at the end of every file, too?

Reply:
resolved in b9c6bf2